src/cmd/cli/command/commands.go

```diff
@@ -620,7 +632,7 @@ var tailCmd = &cobra.Command{
 		term.Info(" * Showing logs since", ts.Format(time.RFC3339Nano), "; press Ctrl+C to stop:")
 		tailOptions := cli.TailOptions{
 			Service: name,
-			Etag:    etag,
+			Etags:   []types.ETag{etag},
```

Comment:
We will allow more than one etag as we want to look at specific deployment events and ECS events (which does not have an etag). For "compose up" the etags will be the deployment etag and "" for service events.